### DISCUSSION TITLE
Can’t find some dub episodes

### DISCUSSION THREAD

**Discussion by Yao-ming23** [Goldenfreddy0703/Otaku#94] [General]
Hi, honestly I'm not sure if I'm posting this in the correct place, so if not please let me know for next time. 

So basically at the moment I'm watching dragon ball super in dub. All good up until ep72, can't find dub, only sub. Am I doing something wrong? I've got real Debrid enabled if that makes a difference, or is it better to not have it? Thanks

**Comment by Goldenfreddy0703** [comment]
Hey so if you already followed my troubleshooting steps on setting up dub then you should be good. There is a possibility that nyaa can get the wrong episodes in a huge series like dragon bass z so you may want to use embeds. With embeds, you can't tell the difference in the quality and it just works. Try embeds and maybe that should help you sense nyaa has trouble scrapping sometimes.

**Comment by Yao-ming23** [comment]
Apologies for the very late response! I'm really noob at GitHub, so apparently I can't find the troubleshooting steps, but I have changed the filter type in settings for "both" sub and dub. 

With regards to embeds, my settings should be playing torrents (I hope I've got it set up correctly like that) but I have tried selecting source and choosing an embed as well as "direct" and for a couple episodes it was the same thing, maybe there just aren't any dubs available for those episodes?

**Reply by Goldenfreddy0703** [reply]
https://github.com/Goldenfreddy0703/Otaku/issues/15

Here are the troubleshooting steps, hopefully this should help you.
